mvx-esdt-safe/tests/mvx_esdt_safe_blackbox_unit_tests.rs

```diff
@@ -106,6 +109,7 @@ fn deposit_too_many_tokens() {
         .check_multiversx_to_sovereign_token_id_mapper_is_empty(TEST_TOKEN_ONE);
 }
 
+/// Test that deposit fails when there is no transfer data
```

Comment:
Deposit should work if there is no transfer data. The name seems to be wrong. Also, the expected error is None for deposit in the test. I don't see anything wrong in this test, it's just a simple successful deposit. Is it duplicated maybe?